frontends/web/src/routes/account/send/send.tsx

```diff
@@ -65,26 +124,26 @@ export default class Send extends Component {
             signConfirm: null, // show visual BitBox in dialog when instructed to sign.
             coinControl: false,
             activeCoinControl: false,
+            data: null,
         };
-        this.selectedUTXOs = [];
     }
 
-    coinSupportsCoinControl = () => {
-        const account = this.getAccount();
-        if (!account) {
+    private coinSupportsCoinControl = () => {
+        if (!this.state.account) {
             return false;
         }
-        return isBitcoinBased(account.coinCode);
+        return isBitcoinBased(this.state.account.coinCode);
     }
 
-    componentDidMount() {
+    public componentDidMount() {
         apiGet(`account/${this.props.code}/balance`).then(balance => this.setState({ balance }));
         if (this.props.deviceIDs.length > 0) {
-            apiGet('devices/' + this.props.deviceIDs[0] + '/has-mobile-channel').then(mobileChannel => {
+            apiGet('devices/' + this.props.deviceIDs[0] + '/has-mobile-channel').then((mobileChannel: boolean) => {
                 apiGet('devices/' + this.props.deviceIDs[0] + '/info').then(({ pairing }) => {
-                    const account = this.getAccount();
+                    const account = this.state.account;
+                    const paired = mobileChannel && pairing;
                     const noMobileChannelError = pairing && !mobileChannel && account && isBitcoinBased(account.coinCode);
-                    this.setState({ paired: mobileChannel && pairing, noMobileChannelError });
+                    this.setState(prevState => ({ ...prevState, paired, noMobileChannelError }));
```

Comment:
Why do we need the `prevState`? Why not just `this.setState({ paired, noMobileChannelError });`?